csharp/ql/src/Stubs/README.md

```diff
@@ -5,10 +5,10 @@ Stubs can be generated from Nuget packages with the `make_stubs_nuget.py` script
 The following calls generate stubs for `Newtonsoft.Json`:
 
 ```
-python make_stubs_nuget.py Newtonsoft.Json
-python make_stubs_nuget.py Newtonsoft.Json latest
-python make_stubs_nuget.py Newtonsoft.Json 13.0.1
-python make_stubs_nuget.py Newtonsoft.Json 13.0.1 /Users/tmp/working-dir
+python3 make_stubs_nuget.py classlib Newtonsoft.Json
```

Comment:
Thanks, I forgot to update this. I think a more natural place for this file (and `make_stubs_nuget.py`) is something like `csharp/scripts/stubs`.